site2/docs/reference-configuration.md

```diff
@@ -4,6 +4,6 @@ title: Pulsar configuration
 sidebar_label: "Pulsar configuration"
 ---
 
-You can manage Pulsar configuration by configuration files in the [`conf`](https://github.com/apache/pulsar/tree/master/conf) directory of a Pulsar [installation](https://pulsar.apache.org/docs/next/getting-started-standalone).
+You can manage Pulsar configuration by configuration files in the `conf` directory of a Pulsar [installation](getting-started-standalone.md).
 
-For a full list of the configuration of different components, see [Pulsar Reference](https://pulsar.apache.org/reference).
+For a full list of the configuration of different components, see [Pulsar Reference](/reference).
```

Comment:
Have you previewed this change? Does the link work?

/ + @SignorMercurio @urfreespace 

/ cc @DaveDuggins @momo-jun

Reply:
That's the right way to write it, I think it will work fine